raiden/network/transport/matrix/transport.py

```diff
@@ -1142,8 +1142,10 @@ def _handle_text(self, room: Room, message: MatrixMessage) -> List[Message]:
             contained all parsed messages is returned.
         """
         assert self._raiden_service is not None, "_raiden_service not set"
-        is_valid_type = (
-            message["type"] == "m.room.message" and message["content"]["msgtype"] == "m.text"
+        is_valid_type = message["type"] == "m.text" or (  # toDevice message
```

Comment:
I believe `toDevice` messages also have `type` as `m.room.message`, and `msgtype` is `m.text`

Reply:
The received message I saw in a test was. The `type` can be changes, but I don't see any `msgtype`
```json
[
    {'content': '{
            "message_identifier": "0",
            "signature": "0xc62e143e02385c86378a6bc3499709bc1eb11db7c07dbd17489f60623083929f77bf19ec91b3631da051e48fdc1877efd66a471e064c115f496e978102eb83161b",
            "type": "Processed"
        }\n{
            "message_identifier": "1",
            "signature": "0x25cf9d643bd68f10037b4cc151a11965177cf2f5bbfb00036cba55fac9722eb169910f265ee348eae1267d7168bd20c3c9522c14408b057b632e6bc1ee4d7e191b",
            "type": "Processed"
        }\n{
            "message_identifier": "2",
            "signature": "0x0e0691fec48b29b78385724620138676e1d9d43a5cc80a112512a0c7374eee793aacddc350520b36cb70824df9804b47eda6a00505ffa55dc2ad283772d4565c1c",
            "type": "Processed"
        }\n{
            "message_identifier": "3",
            "signature": "0x65fbe0c2ca44de8c4eefc7e442171074a8ea38e33dacfb17a14460ba8b1cbbc56afc85b06449a65d0ec737ef91514114bb716427f35848cd4d14b1c0d954bb851c",
            "type": "Processed"
        }\n{
            "message_identifier": "4",
            "signature": "0x4d63999e7e3af262760cbdd42e09821fc47a11a153db3266928cad610f6a67f77ed555ce8b6aebb45fd397897afa4cbba73a58ef5f99ac5a253ca5c937ac3a3e1b",
            "type": "Processed"
        }', 'type': 'm.text', 'sender': '@0xea6e92e5ddc1e108d0ed73045d2a40f268d8ce03:localhost: 8500'
    }
]
```

Reply:
But you have a point. We cannot find out if this is a `m.notice` to start webRTC

Reply:
Not sure, but maybe you may have seen it because synapse os just passing what you sent, and you're sending it like that?
Usually, a toDevice **event** should look like:
```json
{
  "type": "m.room.message",
  "sender": "@0x...",
  "content": {
    "msgtype": "m.text",
    "body": "{\"type\":\"Processed\", ...}"
  }
}
```